content/influxdb/v2.2/reference/cli/influxd/recovery/user/update.md

```diff
@@ -0,0 +1,38 @@
+---
+title: influxd recovery user update
+description: >
+  The `influxd recovery user update` command lets you change your password if you forget your credentials.
```

Comment:
could hypothetically anyone at an org...run `influxdecovery user list`, and then use this command to change another person's password?